core/store/migrate/migrations/0224_create_channel_definition_caches.sql

```diff
@@ -0,0 +1,12 @@
+-- +goose Up
+CREATE TABLE channel_definitions (
+    evm_chain_id NUMERIC(78) NOT NULL,
+    addr bytea CHECK (octet_length(addr) = 20),
```

Comment:
How would upgrading the contract impact the addr is part of the primary key?

Reply:
you'd need to put it behind a proxy then

Reply:
Putting each verifier behind a proxy will increase tx costs while only solving this issue. I suspect there might be a more elegant solution but for now i'll move it to MERC-3652

Reply:
Not sure if it's actually an issue, as the channels will be registered in an old contract. I'll add a note to 3652 for consideration